### PR TITLE
Small layout improvements.

### PR DESCRIPTION
**Pull request by v9ai**
Small POC.

### CODE REVIEW

**Comment by v9ai** [comment]
@benjie what's your thoughts on this ?
Maybe you can add some specific issues since authorization and authentication is not yet hooked on server side ?

File: client/src/components/Layout/Article.js [hunk 1]
```diff
@@ -0,0 +1,8 @@
+import React from "react";
+import styled from "styled-components";
+
+const Article = styled.article`
+  minheight: calc(100vh - 5rem);
```
Comment (benjie) [inline client/src/components/Layout/Article.js]: min-height (missing hyphen)

File: client/src/components/Layout/Footer.js [hunk 2]
```diff
@@ -0,0 +1,9 @@
+import React from "react";
+import styled from "styled-components";
+
+const Footer = styled.footer`
+  padding: 0.5rem 1rem;
+  backgroundcolor: #eee;
```
Comment (benjie) [inline client/src/components/Layout/Footer.js]: Missing hyphen

**Review by benjie — CHANGES_REQUESTED** [review]
LGTM 👍 Just fix the hyphens and we’re good to merge.

**Comment by v9ai** [comment]
Updated.

**Review by benjie — APPROVED** [review]

**Pull request merged**